src/dbus_api/api/prop_conv.rs

```diff
@@ -69,3 +69,51 @@ pub fn locked_pools_to_prop(
         })
         .collect()
 }
+
+/// Convert a stopped pool data structure to a property format.
+pub fn stopped_pools_to_prop(
+    pools: &HashMap<PoolUuid, StoppedPoolInfo>,
+) -> HashMap<String, HashMap<String, Variant<Box<dyn RefArg>>>> {
+    pools
+        .iter()
+        .map(|(u, stopped)| {
+            let mut map = HashMap::new();
+            if let Some(enc_info) = stopped.info.as_ref() {
```

Comment:
"key_description" and "clevis_info" are optional keys, the key, value pairs are only inserted if the pool is encrypted. At present, the stratis-cli PR is written to expect that and intereprets it as revealing whether the pool is encrypted or not. I'm pointing this out, because I want to make sure that it is intentional and intended to be used that way.

Reply:
Yes, that is intended! I couldn't think of a better way to indicate that a pool is encrypted or not with our existing properties.

Reply:
Ok!